environment.yml

```diff
@@ -9,3 +9,8 @@ dependencies:
   # TODO nf-core: Add required software dependencies here
   - fastqc=0.11.8
   - multiqc=1.7
+  - r-base=3.6.1
```

Comment:
Do you think maybe we can change to the most updated R version 4.0?

Reply:
I tried building another docker container with -r-base 4.0.0, which has conflict with the bioconductor packages (dexseq=1.32.0, drimseq=1.14.0, and deseq2=1.26.0) I added on environment.yml.